src/GeneralStateTestsFiller/stSystemOperationsTest/multiSelfdestructFiller.yml

```diff
@@ -0,0 +1,196 @@
+multiSelfdestruct:
+
+  _info:
+    comment: |
+       Implements: SUC000, SUC001, SUC002, SUC003, SUC004, SUC005
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentGasLimit: 0xFF112233445566
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+    currentBaseFee: 1000
+
+
+  pre:
+
+    # Suicidal contract.
+    # Byte 0: What do to
+    #    00 - nothing
+    #    01-FE - transfer that many wei
+    #    FF - self destruct
+    # Bytes 1&2: Recipient address
+    000000000000000000000000000000000000dead:
+      balance: 3
+      nonce: 1
+      code: |
+        :yul {
+           let operation := shr(248, calldataload(0))
+           let recipient := and(shr(232, calldataload(0)), 0xFFFF)
+
+           // Don't do anything
+           if eq(operation, 0) {
+             stop()
+           }
+
+           // Selfdestruct
+           if eq(operation, 0xFF) {
+             selfdestruct(recipient)
+           }
+
+           // Send value
+           // If the call fails, revert
+           if eq(call(gas(), recipient, operation, 0,0, 0,0),0) {
+              revert(0,0)
+           }
+          
+        }
+      storage: {}
+
+
+
+
+    cccccccccccccccccccccccccccccccccccccccc:
+      balance: 100000000
+      nonce: 1
+      code: |
+          :yul {
+             let delme
+
+             // Selfdestruct, send balance to 0x1000
+             // SUC000
+             mstore8(0, 0xFF)
+             mstore8(1, 0x10)
+             mstore8(2, 0x00)
+             delme := call(gas(), 0xdead, 0, 0,3, 0,0)
+             sstore(0x00, balance(0x1000))
+             sstore(0x10, delme)
+
+             // Don't do anything, just receive a wei
+             // SUC001
```

Comment:
No, each test vector assumes the transaction finalization happens.
And then the scenario repeats again.

Otherwise they can entangle and affect each other 

Anyway it is better to put eachvvector in a separate transaction anyway